ui/components/app/home-notification/home-notification.stories.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
This has been extracted to https://github.com/MetaMask/metamask-extension/pull/13035